product_warranty/README.rst

```diff
@@ -0,0 +1,122 @@
+================
+Product Warranty
+================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Production%2FStable-green.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Production/Stable
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Frma-lightgray.png?logo=github
+    :target: https://github.com/OCA/rma/tree/15.0/product_warranty
+    :alt: OCA/rma
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/rma-15-0/rma-15-0-product_warranty
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/145/15.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module extends the functionality of 'Sales Management' to allows you
+to set product warranty details on products and product-supplier relation.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to *Sales > Products > Products (or Product Variants)*
+#. Create a new product (or product variant) or edit an existing one
+   and set 'Warranty Duration' under 'Sales' tab.
+#. If 'Purchase' module is installed, got to
+   *Sales > Products > Products (or Product Variants)*, go to 'Purchase' tab,
+   edit supplier information lines an set the warranty information for each one.
```

Comment:
Can you add a line like this in a file called `readme/CONFIGURATION.rst` with the following contents:

```
To manage the available return instructions, enable debug mode and go to 
*Sales -> Configuration -> Product Return Instructions*.
```

Reply:
Added to `readme/CONFIGURE.rst` ;-)

Reply:
Much better than my suggestion!